spec/models/spree/user_decorator_spec.rb

```diff
@@ -1,27 +1,27 @@
 require "spec_helper"
 
-describe Spree::User do
+describe Spree.user_class, type: :model do
 
   before(:each) do
     @user = FactoryGirl.build(:user_with_loyalty_points)
```

Comment:
@jatin-baweja Instead of defining an instance variable, can we use let! to define user?